qa/run-bandit-tests

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Copyright (c)      2020 Triad National Security, LLC
+#                         All rights reserved.
+#
+# This file is part of the bueno project. See the LICENSE file at the
+# top-level directory of this distribution for more information.
+#
+
+echo "Reporting on Security Standing"
+
+# pip install bandit
```

Comment:
You need to incorporate this dependency in the proper place for GitHub actions; remove this comment. Further, this needs to be added to the suite of tests that are called individually in our CI QA suite.